src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkDemandIndex/index.tsx

```diff
@@ -0,0 +1,135 @@
+import {
+  Clickable,
+  Flex,
+  InfoCircleIcon,
+  Spacer,
+  Text,
+  Tooltip,
+} from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Media } from "Utils/Responsive"
+import { MyCollectionArtworkDemandIndex_marketPriceInsights } from "__generated__/MyCollectionArtworkDemandIndex_marketPriceInsights.graphql"
+import { DemandIndexBar } from "./DemandIndexBar"
+
+interface MyCollectionArtworkDemandIndexProps {
+  marketPriceInsights: MyCollectionArtworkDemandIndex_marketPriceInsights
+}
+
+const DemandIndexExplanation =
+  "Overall strength of demand for this artist and medium combination. Based on the last 36 months of auction sale data from top commercial auction houses."
+
+const MyCollectionArtworkDemandIndex: React.FC<MyCollectionArtworkDemandIndexProps> = ({
+  marketPriceInsights,
+}) => {
+  if (!marketPriceInsights?.demandRank) {
+    return null
+  }
+
+  const demandRank = Number((marketPriceInsights.demandRank * 10).toFixed(2))
+
+  const adjustedDemandRank =
+    demandRank.toFixed(1) === "10.0" ? "9.9" : demandRank
+
+  const isHighDemand = Number(demandRank) >= 9
```

Comment:
**thought:** I think we should probably at some point get the high demand number (9) from metaphysics instead of a hard-typed value as we do now for both app and web. for now, let's maybe have this as a const on the top file to make it easier to find

Reply:
I'm not sure that I completely understand, but I'm assuming that you're suggesting that we should have the adjustment (0.45 => 4.5) in Metaphysics instead of it being on the mobile and web side?